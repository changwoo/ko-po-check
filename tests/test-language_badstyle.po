# -*-text-*-
#
msgid ""
msgstr ""
"Project-Id-Version: test\n"
"POT-Creation-Date: 2012-10-29 02:10+0900\n"
"PO-Revision-Date: 2012-10-26 02:10+0900\n"
"Last-Translator: Changwoo Ryu <cwryu@debian.org>\n"
"Language-Team: Korean <ko@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"

# 불필요한 복수형
msgid "Which ones would you like to save?"
msgstr "어떤 것(들)을 저장할까요?"

# 불필요한 복수형 2 - 알려진 명사
msgid "files"
msgstr "파일들"

# 불필요한 복수형 3 - 들+조사
msgid "do blahblah things this"
msgstr "이것은 뭐시기거시기들을 뭐뭐한다"

# 불필요한 복수형 예외
msgid "turn the handle"
msgstr "핸들을 돌려"

# ...하기 위해
msgid "do something to do what"
msgstr "뭐하기 위해 뭐합니다"

# ...와 함께
msgid "do it with something"
msgstr "무언가와 함께 뭐하십시오"

# ...에 의해
msgid "done by something"
msgstr "무언가에 의해 됐습니다"

# 필요로 하다
msgid "this needs something"
msgstr "이것이 무언가를 필요로 합니다"

# ...할 것입니다
msgid "this will do something"
msgstr "무언가를 해버릴 것입니다"

# 예외: 과거형
msgid "this will do something"
msgstr "무언가가 잘못됐을 것입니다"

# ...하시기 바랍니다
msgid "Please do blahblah"
msgstr "블라블라 하시기 바랍니다"

# 몇개의 아무개
msgid "Copying %d files"
msgstr "%d개의 파일을 복사 중"

# ...을 포함
msgid "It has something containing another"
msgstr "다른 무언가를 포함하는 무언가가 있습니다"
