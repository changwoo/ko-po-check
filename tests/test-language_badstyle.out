-:16: 것(들): 불필요한 복수형 표현
-:20: 파일들: 불필요한 복수형 표현
-:24: 뭐시기거시기들을 : 불필요한 복수형 표현
-:32: 뭐하기 위해: 어색한 표현. "-하려면"과 같이 쓰십시오.
-:36: 무언가와 함께: 어색한 표현. 풀어서 써 보십시오.
-:40: 무언가에 의해: 어색한 표현. 수동태 문장을 바꿔 보십시오.
-:48: 해버릴 것입니다: 어색한 표현. "...합니다"와 같이 간결히 쓰십시오.
-:56: 하시기 바랍니다: 어색한 표현. "...하십시오"와 같이 간결히 쓰십시오.
-:60: %d개의 파일을: 어색한 표현. "몇개의 아무개" 대신 "아무개 몇개"와 같이 쓰십시오.
-:64: 무언가를 포함하는: 어색한 표현. "..이 들어 있다"와 같이 바꿔 쓰십시오.
